### PR TITLE
Adjusted Founder Turnover Modifier & Updated Documentation

### PR DESCRIPTION
**Pull request by IllianiCBT**
Requires #4101

The turnover target number modifier for Founders has been buffed from +2 to +4.

Documentation has been updated to include this and the introduction of a 'Pay Bonus Default' setting that was accidentally pushed to #4101.

### CODE REVIEW

File: MekHQ/src/mekhq/campaign/personnel/enums/PersonnelStatus.java [hunk 1]
```diff
@@ -224,6 +246,38 @@ public static PersonnelStatus parseFromString(final String text) {
...
+                case 17:
+                    return OLD_AGE;
+                case 18:
+                    return MEDICAL_COMPLICATIONS;
+                case 19:
+                    return PREGNANCY_COMPLICATIONS;
+                case 20:
+                    return UNDETERMINED;
+                case 21:
+                    return SUICIDE;
```
Comment (SJuliez) [inline MekHQ/src/mekhq/campaign/personnel/enums/PersonnelStatus.java:280]: ever play Rimworld? There's also FOOD_BINGE and TANTRUM
:D

File: MekHQ/src/mekhq/campaign/unit/Unit.java [hunk 2]
```diff
@@ -4732,6 +4732,30 @@ public void cancelMothballOrActivation() {
     }
     //endregion Mothballing/Activation
 
+    /**
+     * Returns a list of all soldiers in a unit.
+     *
+     * @return a list of Person objects representing the soldiers.
+     */
+    public List<Person> getSoldiers() {
```
Comment (SJuliez) [inline MekHQ/src/mekhq/campaign/unit/Unit.java:4740]: I suggest giving these methods good titles and/or comprehensive comments. We have many such methods in MM for equipment (hasWorkingMisc...) and it's never quite clear from title/comment what they really do. Might e.g. place the // comments into the javadoc. And, maybe state that soldiers are Infantry personnel (which includes BA)
Also, I believe in a case like this, the @return text alone would be alright. No need to repeat it as in l.4736 unless there's additional info somewhere.

**Review by SJuliez — APPROVED** [review]

File: MekHQ/src/mekhq/campaign/personnel/enums/PersonnelStatus.java (hunk 1 above)
Comment (IllianiCBT) [reply MekHQ/src/mekhq/campaign/personnel/enums/PersonnelStatus.java:280]: Ironically, given I've become the MechSims guy, I don't actually _like_ playing games like Rimworld or Dwarf Fortress. Though I do enjoy listening to people recount their Dwarf Fortress misadventures.

**Comment by IllianiCBT** [comment]
Addressed requested changes

**Pull request merged**